### PR TITLE
Prep 0.4.0 release

### PR DESCRIPTION
**Pull request by vtnate**

### CODE REVIEW

**Comment by nllong** [comment]
oh no... failing test makes me sad...

**Comment by vtnate** [comment]
@nllong Let's not get too sad yet. Many of those are canceled, not failing, because I pushed something new on top of them. The most recent one looks like a github actions hiccup. I'll re-run the failed tests and they should pass.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,30 @@
 Change Log
 ==========
 
+Version 0.4.0
+-------------
+As of version 0.4.0 changes will be published using Github automated formatting at the release itself. Those changes are copied here.
```
Comment (nllong) [inline CHANGELOG.rst:6]: eggsactly.

**Review by nllong — APPROVED** [review]
let's go!

**Pull request merged**